codeforamerica/vita-min
# add new id state logo and adjust state logo css to better fit design and keep id logo legible

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1476
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Base64 encoded the provided Idaho logo png and embedded it in an svg
- Simplified the Idaho logo svg to decrease the number of required magic numbers and make the png render closer to native resolution (it's less fuzzy now)
- Adjusted the state logo css to have a larger max-width, which doesn't affect most states but makes the Idaho logo more legible
## How to test?
- Look at the ID landing page and verify that the new logo appears and looks fine
- Look at the other state landing pages and make sure they still look fine
## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/c772b7e4-d8b6-4982-a304-451608f33fb2)
